exercises/clock/README.md

```diff
@@ -18,11 +18,24 @@ add an `Eq` instance, and implement the functions:
 `addDelta` adds a duration, expressed in hours and minutes, to a given time,
 represented by an instance of `Clock`.
 
+`fromHourMin` takes an hour and minute, and returns an instance of `Clock` with 
+those hours and minutes.
+
+`toString` takes an instance of `Clock` and returns a string representation 
+of the clock, in 0-padded format like "08:03" or "22:35"
+
 You will find a dummy data declaration and type signatures already in place,
 but it is up to you to define the functions and create a meaningful data type,
 newtype or type synonym.
 
+If you need help getting started with Types, take a look at:
+- [Making Our Own Types and Typeclasses][types]
+
+[types]: http://learnyouahaskell.com/making-our-own-types-and-typeclasses
+
+## Restrictions
 
+Don't use the Data.Dates package, create the functions yourself!
```

Comment:
> One quick question though, what is your comment on restrictions?

So that comment disappeared because of a bug in GitHub, it seems.

I'll try to repeat it.

You may be confusing "package" and "module". I'm not sure exactly what package or module you're referring to, since the most used date/time package, `time`, has the module `Data.Time.Clock`, and [various other popular time libraries](https://jacobstanley.io/3-packages-you-need-to-know-about-before-processing-timestamps-in-haskell/) have different ones.

I'd get rid of this restriction because using library functions [encourages fluency](https://exercism.io/values). Alternatively, if mentors report that a majority of students solve this exercise in a too simple manner, one could consider deprecating it like "gigasecond" was deprecated in #280 (see argument in #230).

```suggestion
(remove this; I cannot seem to suggest emptiness)
```

Reply:
Sounds good, I like that philosophy